binding-mruby/mrb-ext/kernel.cpp

```diff
@@ -172,9 +172,10 @@ MRB_FUNCTION(kernelLoadData)
 	mrb_get_args(mrb, "z", &filename);
 
 	SDL_RWops ops;
-	GUARD_EXC( shState->fileSystem().openRead(ops, filename); )
+	GUARD_EXC( shState->fileSystem().openReadRaw(ops, filename); )
 
 	mrb_value obj;
+
```

Comment:
no spurious whitespace changes pls

Reply:
Sorry about this. Perils of working late night =|